File: Programacion-Estructurada-1-1/algoritmo.py
def ejercicio():
    acumulador = 0
    contador = 0
    edad = float( input("Ingresar Edad: ") )

    while ( edad != 0 ):
        acumulador += edad
        contador += 1
        edad = float( input("Ingresar Edad: ") )

    promedio = acumulador/contador
    print(f"El promedio es: {promedio}")

ejercicio()

File: Programacion-Estructurada-1-1/algoritmo.c
#include <stdio.h>

void ejercicio() {
    float acumulador;
    float contador;
    float edad;
    float promedio;
    contador = 0;
    acumulador = 0;
    printf("Ingresar Edad: ");
    scanf("%f", &edad);

    while ( edad != 0) {
        acumulador = acumulador + edad;
        contador = contador + 1;
        printf("Ingresar Edad: ");
        scanf("%f", &edad);
    }

    promedio = acumulador/contador;
    printf("El promedio es: %f", promedio);
}

int main()
{ 
    ejercicio();
    return 0;
}